src/main/java/uk/ac/ebi/subs/fileupload/services/EventHandlerService.java

```diff
@@ -12,4 +16,18 @@
     ResponseEntity<Object> validateUploadRequest(TUSFileInfo tusFileInfo);
 
     ResponseEntity<Object> persistOrUpdateFileInformation(File file);
+
+    void moveFile(File file) throws IOException;
```

Comment:
I'm not sure this needs to be in the shared service, looks like it is only needed in one place

Reply:
OK. I will have a look and refactor it